templates/benchmark.html

```diff
@@ -34,69 +36,127 @@ <h4>potential anomalies</h4>
 </table>
 {% endfor %}
 
-<h3>overall branch predict ratio</h3>
-<canvas id="branch-predict-ratio-chart" width="600" , height="200"></canvas>
+<a id="branch-predict-ratio"></a>
+<h3><a href="#branch-predict-ratio">*</a> overall branch predict ratio</h3>
+<div id="branch-predict-ratio-chart" style="width: 100%; height: 600px"></div>
 
-<h3>overall cache hit ratio</h3>
-<canvas id="cache-ratio-chart" width="600" , height="200"></canvas>
+<a id="cache-ratio"></a>
+<h3><a href="#cache-ratio">*</a> overall cache hit ratio</h3>
+<div id="cache-ratio-chart" style="width: 100%; height: 600px"></div>
 
+<script src="https://cdn.jsdelivr.net/npm/highcharts@6/highcharts.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/highcharts@6/highcharts-more.min.js"></script>
 
 <script>
+    const means = JSON.parse(`{{means|json}}`);
+    const stdDevs = JSON.parse(`{{std_devs|json}}`);
     const anomalyData = JSON.parse(`{{anomalous_timings|json}}`);
     const chartData = JSON.parse(`{{timings|json}}`);
     chartData.reverse();
 
-    let labels = chartData.map(t => t.toolchains[0].spec);
+    const labels = chartData.map(t => new Date(t.toolchains[0].spec.replace('nightly-', '')).valueOf());
```

Comment:
I'm not sure how best to approach this, but pretty soon we'll need to have charts with many different non-nightly toolchains still presented in the order in which they occurred (see https://github.com/anp/lolbench/issues/29#issuecomment-426861774 for an example). I think this is fine for now, could you leave a comment in the code explaining that this will need to change?